File: Essential Algorithms/Lab 1/Q3.py
def dsum(n):
    if n<1:
        print("Please input an non-negative number")
    else:
        summation = 0
        i=1
        j=1
        while i<=n:
            while j<=n:
                summation += 3*i
                j+=1
            i+=1
            j=1
        print("Answer for",n,":",summation)

dsum(3)

File: Essential Algorithms/Lab 1/Q3.js
function dsum(n)
{
    if (n<1)
    {
        console.log("Please input a non-negative number");
    }
    else
    {
        let summation = 0;
        let i = 1;
        let j = 1;
        while (i<=n)
        {
            while (j<=n)
            {
                summation += 3*i;
                j+=1;
            }
        i+=1;
        j=1;
        }
    console.log("Answer for",n,":",summation);
    }
}

dsum(3);